ReportSimulationOutput/measure.rb

```diff
@@ -353,6 +365,28 @@ def energyPlusOutputRequests(runner, user_arguments)
       if args[:include_timeseries_fuel_consumptions]
         result << OpenStudio::IdfObject.load("Output:Meter,ElectricStorage:ElectricityProduced,#{args[:timeseries_frequency]};").get
       end
+
+      # Resilience
+      if args[:include_annual_resilience] || args[:include_timeseries_resilience]
+        resilience_frequency = 'timestep'
+        if args[:timeseries_frequency] != 'timestep'
+          resilience_frequency = 'hourly'
+
+          # We need the following request in case no other output types are requested
+          # I.e., this will force a eplusout_xxx.msgpack from which we get timestamps
+          if ['daily', 'monthly'].include?(args[:timeseries_frequency])
+            result << OpenStudio::IdfObject.load("Output:Variable,*,Site Outdoor Air Drybulb Temperature,#{args[:timeseries_frequency]};").get
+          end
```

Comment:
Hmm. I don't immediately understand why we need this for resilience outputs but not for emissions outputs. How does it work when you request daily/monthly emissions outputs?

Reply:
@shorowit I think it may be broken for when you request monthly emissions (and no other timeseries outputs). Try:

```
$ /c/openstudio-3.6.1/bin/openstudio.exe workflow/run_simulation.rb -x workflow/sample_files/base-pv-battery-scheduled.xml --monthly emissions
```

and you don't see either eplusout_monthly.msgpack or results_timeseries.csv files.

Reply:
@joseph-robertson That's because there are no emissions scenarios defined in that HPXML. If you copy emissions scenarios from base-misc-emissions.xml into it and then try again, you'll get monthly emissions outputs.

Reply:
Here's my solution (similar to what is done for emissions):
```
  def setup_timeseries_includes(emissions, args)
    # To calculate timeseries emissions or timeseries fuel consumption, we also need to select timeseries
    # end use consumption because EnergyPlus results may be post-processed due to HVAC DSE.
    # TODO: This could be removed if we could account for DSE inside EnergyPlus.
    args = args.dup # We don't want to modify the original arguments
    args[:include_hourly_electric_end_use_consumptions] = false
    if not emissions.empty?
      args[:include_hourly_electric_end_use_consumptions] = true # Need hourly electricity values for Cambium
      if args[:include_timeseries_emissions] || args[:include_timeseries_emission_end_uses] || args[:include_timeseries_emission_fuels]
        args[:include_timeseries_fuel_consumptions] = true
      end
    end
    if args[:include_timeseries_total_consumptions] || args[:include_timeseries_resilience] <== ADDED
      args[:include_timeseries_fuel_consumptions] = true
    end
    if args[:include_timeseries_fuel_consumptions]
      args[:include_timeseries_end_use_consumptions] = true
    end
    if args[:include_timeseries_system_use_consumptions]
      args[:include_timeseries_end_use_consumptions] = true
    end
    return args
  end
```

Reply:
Thanks Joe, that looks good. If you remove the ADDED code, do you know if any of our CI tests will fail? I'd like to ensure we have a test that covers this situation.